build-logic-commons/build-platform/build.gradle.kts

```diff
@@ -69,7 +72,7 @@ dependencies {
             because("CVE-2022-40152 on lower versions")
         }
         api("com.beust:jcommander:1.78")
-        api("org.codehaus.groovy:groovy:$groovyVersion")
+        api("$groovyGroup:groovy:$groovyVersion")
         api("org.codehaus.groovy.modules.http-builder:http-builder:0.7.2") // TODO maybe change group name when upgrading to Groovy 4
```

Comment:
💅 Looks like `org.codehaus.groovy.modules.http-builder:http-builder:0.7.2` is no longer relevant, should it be removed as part of the upgrade?

Reply:
Yeah, this is still using Groovy 3 so I wasn't sure, but taking a closer look it seems all of those classes aren't used in our projects, so it can probably be removed.